cg/utils/files.py

```diff
@@ -8,6 +8,10 @@
 LOG = logging.getLogger(__name__)
 
 
+def get_project_root_dir() -> Path:
+    return Path(__file__).parent.parent
```

Comment:
This is also an option, but I'm not sure which one is best in this context. I ended up using the `importlib.resources` module for the SWEDAC delivery report logo path

```suggestion
from importlib.resources import files

def get_project_root_dir() -> Path:
    return files("cg")
```

Reply:
I think your is better as it avoids the dunders access. Updated. @Vince-janv Do you know a good way to test the invoice part?